pychunkedgraph/graph/chunkedgraph.py

```diff
@@ -116,10 +116,15 @@ def range_read_chunk(
         )
 
     def get_atomic_id_from_coord(
-        self, x: int, y: int, z: int, parent_id: np.uint64, n_tries: int = 5
+        self,
+        x: int,
+        y: int,
+        z: int,
+        parent_id: typing.Optional[np.uint64] = None,
```

Comment:
where is `get_atomic_id_from_coord` called with `parent_id=None`?
rest looks good

Reply:
You're right, it seems that was a testing thing that is no longer needed. I removed it, thanks for identifying